tox.ini

```diff
@@ -1,22 +1,20 @@
 [tox]
 envlist =
-    py{27,34,35,36}-django-111
-    py{34,35,36}-django-20
-    py{35,36}-django-master
+    py{27,34,35,36,37}-django-111
+    py{34,35,36,37}-django-20
+    py{35,36,37}-django-21
+    py{35,36,37}-django-master
 
 [testenv]
 deps =
-    coveralls
-    django-appconf
-    django-ipware
+    -r requirements.txt
     django-111: django>=1.11,<2.0
     django-20: django>=2.0,<2.1
+    django-21: django>=2.1,<2.2
     django-master: https://github.com/django/django/archive/master.tar.gz
-    astroid==1.6.5 
-    prospector==0.12.11
-    py27: mock
 usedevelop = True
 ignore_outcome =
+    django-21: True
```

Comment:
Issue #374 is related to this change: I just added this so that we would have explicit testing against the LTS as well. We already had `django-master` with the same `ignore_outcome` rule running here.